src/Fixer/Operator/OperatorLinebreakFixer.php

```diff
@@ -94,6 +107,7 @@ public function configure(array $configuration = null)
             }
         }
         $this->position = $this->configuration['position'];
```

Comment:
Can we refactor `$this->position`, sot the keys are the operator contents, so we don't need `$this->ignoredOperators` and can use simply `array_diff`?